scripts/dbt/project.py

```diff
@@ -1,47 +0,0 @@
-import os
-from SCons.Script import GetOption
-from dbt.util import vcheck, vprint
-
-
-def compose_valid_targets(cmd_env=None):
```

Comment:
Is this file just unused? It doesn't seem related to tests

Reply:
yeah, this is for the old SCons build. I just noticed it while adding the build task, it should probably technically be a separate PR...